### PR TITLE
QAWTO-214: update appium python client dependency

### PR DESCRIPTION
**Pull request by hesteban-tuenti**
As part of the maintenance of the toolium dependency libraries, the update of the appium python client is going to be update to the version 4.
In order to update to this version we identified that the following actions needed to be updated to be compatible with the w3C webdriver:
https://jira.tid.es/browse/QAWTO-178

In order to be fully W3C compatible we need to change the set_value method used in the InputText pagelement to send_keys

### CODE REVIEW

**Comment by hesteban-tuenti** [comment]
reviso los tests cuando tenga un momento

File: toolium/pageelements/input_text_page_element.py [hunk 1]
```diff
@@ -26,22 +26,18 @@ def text(self):
 
         :returns: element text value
         """
-        if self.driver_wrapper.is_web_test() or self.webview:
-            return self.web_element.get_attribute("value")
-        elif self.driver_wrapper.is_ios_test():
-            return self.web_element.get_attribute("label")
```
[the remark this answers is not in the document]
Comment (rgonalo) [reply toolium/pageelements/input_text_page_element.py:32]: si funciona con .get_attribute("value") déjalo así, pero tienes que actualizar los tests unitarios para que no fallen

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 requests~=2.27            # api tests
 selenium~=4.0             # web tests
-Appium-Python-Client~=2.3 # mobile tests
+Appium-Python-Client==4.0.1 # mobile tests
```
Comment (rgonalo) [inline requirements.txt:3]: no deberíamos tener una versión fija, sólo la mínima, debería ser por ejemplo
Appium-Python-Client~=4.0

**Review by rgonalo — CHANGES_REQUESTED** [review]

**Comment by hesteban-tuenti** [comment]
I have tested the new mobile_actions features (Android e iOS) which tests all of the detected changes in Appium 3 , and 4 and also the change in the InputText done in this PR with the following versions:

 * Appium-Python-Client= 2.11.1 - Selenium=4.11.2
 * Appium-Python-Client= 3.2.1 - Selenium=4.25
 * Appium-Python-Client= 4.2.1  Selenium=4.25

And the results have been OK

File: README.rst [hunk 3]
```diff
@@ -30,6 +30,9 @@ The main dependencies are:
   in Android or iOS devices/emulators.
 - `requests <http://docs.python-requests.org>`_: to test APIs
 
+You might need to adjust the Selenium and Appium-Python-Client versions in your project.
```
Comment (hesteban-tuenti) [inline README.rst:33]: WDYT?

**Review by rgonalo — DISMISSED** [review]

**Pull request merged**